holdings/src/data.json

```diff
@@ -0,0 +1,58 @@
+[{
```

Comment:
the holdings service is weird.. mainly because it returns users, not holdings. I think we should rename it to users service and rename the routes to `/users` instead of `/holdings`. It's the quickest thing to do to fix it. Otherwise we'll have to rethink how the routes work in a clean way. For example the `holdings/:userId` route shouldn't exist in real life. Here's a list of acceptable routes:
-/users
- /users/:userId
- /users/:userId/holdings
- /users/:userId/holdings/:holdingsId

Reply:
really it's /userHoldings/:userId

Reply:
I was thinking about this when I wrote it, really it's a filter so it would probably be better as `holdings?userId=:userId`.
We could just remove it all together, I only put it in to add some ramda into the codebase as an example

Reply:
So going back to the original /holdings/id but use ramda to do the find even if it's a bit unecessary

Reply:
So my thought is if we just rename the service and routes to `users` rather than `holdings` we wouldn't need to change anything else